1주차/5622.py

```diff
@@ -0,0 +1,28 @@
+#5622
+##걸린 시간 = 다이얼 숫자+1
+
+A=input("")
+B=list(A)
+num=0
+sum=0
+for i in range(0,len(B)):
```

Comment:
```suggestion
for i in range(len(B)):
```
range 0부터 시작할때는 시작 숫자 안적어줘도 됩니다